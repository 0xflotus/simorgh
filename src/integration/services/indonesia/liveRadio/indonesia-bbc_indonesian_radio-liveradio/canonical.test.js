/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service indonesia
 * @pathname /indonesia/bbc_indonesian_radio/liveradio
 */

import runCanonicalTests from '../../../../pages/liveRadio/canonicalTests';

describe('Canonical indonesia liveRadio', runCanonicalTests);
