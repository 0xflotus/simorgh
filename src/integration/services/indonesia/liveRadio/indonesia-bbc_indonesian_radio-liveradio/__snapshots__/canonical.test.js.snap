// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical indonesia liveRadio A11y I can see the skip to content link 1`] = `"Langsung ke konten"`;

exports[`Canonical indonesia liveRadio Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=54&amp;p=indonesia.bbc_indonesian_radio.liveradio.page&amp;x1=[urn:bbc:ares:ws_media:live:bbc_indonesian_radio]&amp;x2=[responsive]&amp;x3=[news-indonesia]&amp;x4=[id]&amp;x7=[player-live]&amp;x8=[simorgh-nojs]&amp;x9=[BBC+Indonesia+Radio+-+BBC+News+Indonesia]\\"/>"`;

exports[`Canonical indonesia liveRadio Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/indonesia"`;

exports[`Canonical indonesia liveRadio Footer I can see the footer copyright and external linking text 1`] = `"Â© 2020 BBC. BBC tidak bertanggung jawab atas konten dari situs eksternal. Baca tentang peraturan baru terkait link eksternal."`;

exports[`Canonical indonesia liveRadio Header I can see the branding 1`] = `"BBC News, Indonesia"`;

exports[`Canonical indonesia liveRadio I can see the headline 1`] = `"BBC Indonesia Radio"`;

exports[`Canonical indonesia liveRadio I can see the summary 1`] = `"BBC Indonesia Radio: Berita dunia, ulasan, berita olahraga dan majalah mingguan dari BBC Indonesia Radio."`;

exports[`Canonical indonesia liveRadio Media player embed A11y I can read the media player title 1`] = `"Pemutar Audio"`;

exports[`Canonical indonesia liveRadio Media player embed I can see the media player embed 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/media/bbc_indonesian_radio/liveradio/id?morph_env=live"`;

exports[`Canonical indonesia liveRadio SEO Canonical link 1`] = `"http://localhost:7080/indonesia/bbc_indonesian_radio/liveradio"`;

exports[`Canonical indonesia liveRadio SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical indonesia liveRadio SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical indonesia liveRadio SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical indonesia liveRadio SEO Heading level 1 1`] = `"BBC Indonesia Radio"`;

exports[`Canonical indonesia liveRadio SEO Lang attribute 1`] = `"id"`;

exports[`Canonical indonesia liveRadio SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "RadioChannel",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/indonesia.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/indonesia/bbc_indonesian_radio/liveradio",
    "@type": "WebPage",
    "name": "BBC Indonesia Radio",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/indonesia.png",
      "width": 1024,
    },
    "name": "BBC News Indonesia",
    "publishingPrinciples": "https://www.bbc.com/indonesia/institutional-49283175",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/indonesia.png",
  "url": "http://localhost:7080/indonesia/bbc_indonesian_radio/liveradio",
}
`;

exports[`Canonical indonesia liveRadio SEO OG description 1`] = `"BBC Indonesia Radio: Berita dunia, ulasan, berita olahraga dan majalah mingguan dari BBC Indonesia Radio."`;

exports[`Canonical indonesia liveRadio SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/indonesia.png"`;

exports[`Canonical indonesia liveRadio SEO OG image alt 1`] = `"BBC News Indonesia"`;

exports[`Canonical indonesia liveRadio SEO OG locale 1`] = `"id-ID"`;

exports[`Canonical indonesia liveRadio SEO OG site name 1`] = `"BBC News Indonesia"`;

exports[`Canonical indonesia liveRadio SEO OG title 1`] = `"BBC Indonesia Radio - BBC News Indonesia"`;

exports[`Canonical indonesia liveRadio SEO OG type 1`] = `"website"`;

exports[`Canonical indonesia liveRadio SEO OG url 1`] = `"http://localhost:7080/indonesia/bbc_indonesian_radio/liveradio"`;

exports[`Canonical indonesia liveRadio SEO Page title 1`] = `"BBC Indonesia Radio - BBC News Indonesia"`;

exports[`Canonical indonesia liveRadio SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical indonesia liveRadio SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical indonesia liveRadio SEO Twitter creator 1`] = `"@bbcindonesia"`;

exports[`Canonical indonesia liveRadio SEO Twitter description 1`] = `"BBC Indonesia Radio: Berita dunia, ulasan, berita olahraga dan majalah mingguan dari BBC Indonesia Radio."`;

exports[`Canonical indonesia liveRadio SEO Twitter image alt 1`] = `"BBC News Indonesia"`;

exports[`Canonical indonesia liveRadio SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/indonesia.png"`;

exports[`Canonical indonesia liveRadio SEO Twitter site 1`] = `"@bbcindonesia"`;

exports[`Canonical indonesia liveRadio SEO Twitter title 1`] = `"BBC Indonesia Radio - BBC News Indonesia"`;
